models/dex/dex_info.sql

```diff
@@ -72,7 +72,6 @@ FROM (VALUES
     , ('0x API', '0x API', 'Aggregator', '0xProject') -- To Fix, should be just zeroex
     , ('paraswap', 'ParaSwap', 'Aggregator', 'paraswap')
     , ('cow_protocol', 'CoW Swap', 'Aggregator', 'CoWSwap')
```

Comment:
This name could also be changed to CoWSwap if need be.

Reply:
ah is `CoWSwap` the proper name and not `CoW Swap`? i want to make sure it's as accurate as possible

Reply:
I guess technically it's two words so can be left as is. Just thought you were aiming to remove spaces etc

Reply:
But if you look two lines above, I still see "0x API" as the project field.

Reply:
Yeah project field should all me snake_case but some like 0x remain to be fixed, will do that in other PRs. The name column in .info should be the properly capitalised & spaced name just to use for visualisations